alexberka/a-trivial-glance
# Host drag and drop + styling

## Description
Host users can adjust question status directly on /host/game/[id] page through drag and drop between columns
Also added buttons to /host/game/[id] to close an open question or release answers for all closed questions

Regrouped page into header/content divs to improve styling control
Added bg elements to and styled sign-in page

Fixed a couple undefined data as they presented

## Related Issue
NA

## Motivation and Context
As a host user, it was tedious to navigate to a gameQuestion page to toggle the game status. So I wanted tools to change a question's status directly on the game display

Styling changes will make media queries/responsiveness easier to manage

## How Can This Be Tested?
Navigate to a game's details page as a host. When the game is closed, most drag and drop functionality (apart from resetting questions) is locked. With game live, move questions between the four panels and check that behavior is as expected. Open a second panel as a player in that game to view effects of drag and drop.

## Screenshots (if appropriate):

## Types of changes
